### ISSUE TITLE
Test failure (due to GHC 8.2.2?)

### ISSUE THREAD

**Issue by bergmark** [fpco/weigh#15]
Getting this while building nightly. I'm not sure if I should remove weigh from the 8.2.2 nightly or just disable the test suite, what do you think? I also can't guarantee it's a GHC-8.2.2 specific issue, but this didn't appear until the upgrade.

```
> /tmp/stackage-build12/weigh-0.0.5$ dist/build/weigh-test/weigh-test

Case                             Allocated            GCs
integers count 0                        16  4,294,967,296
integers count 1                        32  4,294,967,296
integers count 2                        48  4,294,967,296
integers count 3                        64  4,294,967,296
integers count 10                      176  4,294,967,296
integers count 100                   1,616  4,294,967,296
integers count IO CAF 0                 32  4,294,967,296
integers count IO func 0                40  4,294,967,296
integers count IO CAF 1                 48  4,294,967,296
integers count IO func 1                56  4,294,967,296
ints count 1                            16  4,294,967,296
ints count 10                           16  4,294,967,296
ints count 1000000                      16  4,294,967,296
\_ -> IntegerStruct 0 0                 16  4,294,967,296
\x -> IntegerStruct x 0                 40  4,294,967,296
\x -> IntegerStruct x x                 40  4,294,967,296
\x -> IntegerStruct (x+1) x             56  4,294,967,296
\x -> IntegerStruct (x+1) (x+1)         56  4,294,967,296
\x -> IntegerStruct (x+1) (x+2)         72  4,294,967,296
\x -> HasInt x                          32  4,294,967,296
\x -> HasUnpacked (HasInt x)            32  4,294,967,296
\x -> HasPacked (HasInt x)              72  4,294,967,296

Check problems:
  ints count 1
    Allocated bytes exceeds 0: 16
  ints count 10
    Allocated bytes exceeds 0: 16
  ints count 1000000
    Allocated bytes exceeds 0: 16
```

**Comment by chrisdone** [comment]
Please disable it from the build for now -- it needs updating for GHC 8.2.2 due to https://github.com/fpco/weigh/issues/13#issuecomment-344592122.

**Comment by chrisdone** [comment]
Fixed in weigh-0.0.6. :metal:
